modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -185,7 +185,7 @@ def sort_recent_appointments(appointments)
             Rails.logger.info("VAOS appointment sorting filtered out id #{rem_appt.id} due to missing start time.")
           end
         end
-        filtered_appts.sort_by { |appointment| DateTime.parse(appointment.start) }
+        filtered_appts.sort_by { |appointment| DateTime.parse(appointment.start) }.reverse
```

Comment:
FYI previously this was sorted in ascending order which meant oldest appointment first which I think is wrong so I'm updating it to be in descending order.